### PR TITLE
Integrated Marshal for client send.

### PR DESCRIPTION
**Pull request by thelimeburner**

### CODE REVIEW

**Comment by amygara** [comment]
Need to pull from origin/client-dev

File: client/client-kv.c [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+#include <string.h>
+#include "../lib/marshal/marshal.h"
+/****************************************
+ Author: Joel Klein, Katie Stasaski, Lucas Chaufournier, Tim Wood
+ with a little help from
+ http://beej.us/guide/bgnet/
+ ****************************************/
+//Returns the number of bytes to wait for
+
+struct operation* read_line(int sockfd){
```
Comment (amygara) [inline client/client-kv.c]: For all following function prototypes, just be sure to adhere to the style guide

**Comment by phil-lopreiato** [comment]
Build fails. Why?

**Comment by phil-lopreiato** [comment]
See 

```
make[1]: Entering directory `/home/travis/build/gwAdvNet2015/gw-kv-store/client'
make[1]: *** No rule to make target `test'.  Stop.
```

You're going to want to add a `test` target (you don't need to actually write tests, you can just echo "No tests here" for now) into `client/Makefile`

**Comment by phil-lopreiato** [comment]
Made to wrong branch. Superseded by #46

**Pull request closed without merging**